dcpy/test/extract/test_utils.py

```diff
@@ -0,0 +1,18 @@
+from dcpy.connectors.edm import recipes
+from dcpy.extract import utils
+
+
+class TestSocrata:
```

Comment:
This makes a callout to Socrata? I don't think it'll be problematic, but I suppose it's conventional to separate out these potentially flaky tests.